lib/src/create/presenter/bloc/create_post_bloc.dart

```diff
@@ -11,6 +12,20 @@ class CreatePostBloc extends Bloc<CreatePostEvent, CreatePostState> {
 
   CreatePostBloc() : super(const CreatePostInitial()) {
     on<PostTextChanged>(_onTextChanged);
+    on<GifSelected>((event, emit) {
```

Comment:
Para mantener una consistencia en el código y con solo un método manejar si un gif fue seleccionado o eliminado se podría cambiar por lo siguiente:

```suggestion
    on<PostGifChanged>(_onGifChanged);
```